### PR TITLE
Tutorial.rst: Add 'noqa' and examples

### LINKED ISSUE #366: Add 'noqa' to documentation

### PR DESCRIPTION
**Pull request by RohanVB**
Added details about the keyword 'noqa'
along with related examples. It has the
same functionality as 'ignore'

Closes #366

### CODE REVIEW

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -323,6 +323,23 @@ If you put an ``all`` instead of the bear names directly after the
...
+::
+    # noqa 
+        long_line = "This is a long line, long enough to trigger LineLengthBear"       
+
+If you wish to specify which bear to use with noqa, as is done with 'Ignore', 
+we would proceed as follows: 
+
+::
+    # noqa LineLengthBear
+         long_line = "This is a long line, long enough to trigger LineLengthBear"
```
Comment (sims1253) [inline Users/Tutorial.rst]: same here

File: Users/Tutorial.rst [hunk 2]
```diff
@@ -323,6 +323,23 @@ If you put an ``all`` instead of the bear names directly after the
 ``ignore``/``ignoring`` keyword, the results of all bears affecting
 those lines will be ignored.
 
+If you've used another linter in the past, you don't have to change your
+pre-existing code with the 'noqa' keywords to 'ignore' as the examples below
```
Comment (Asnelchristian) [inline Users/Tutorial.rst]: maybe noqa  should be within backsticks

File: Users/Tutorial.rst [hunk 3]
```diff
@@ -323,6 +323,23 @@ If you put an ``all`` instead of the bear names directly after the
...
+work as well. If no bears are specified, noqa will be applicable to work for 
+all bears. 
+
+::
+    # noqa
+        long_line = "This is a long line ... "       
+
+If you wish to specify which bear to use with noqa, as is done with 'Ignore',
+we would proceed as follows:
```
Comment (Asnelchristian) [inline Users/Tutorial.rst]: ``you should proceed as follows`` i think

File: Users/Tutorial.rst [hunk 4]
```diff
@@ -323,6 +323,23 @@ If you put an ``all`` instead of the bear names directly after the
...
 those lines will be ignored.
 
+If you've used another linter in the past, you don't have to change your
+pre-existing code with the 'noqa' keywords to 'ignore' as the examples below
+work as well. If no bears are specified, noqa will be applicable to work for 
+all bears. 
+
+::
+    # noqa
+        long_line = "This is a long line ... "
```
Comment (sils) [inline Users/Tutorial.rst]: why does the indentation differ?

File: Users/Tutorial.rst [hunk 5]
```diff
@@ -317,12 +317,31 @@ start with `Line`, `Py`, and `Px`.
 ::
 
     # Ignore LineLengthBear and SpaceConsistencyBear
-        variable = "Why the heck are spaces used instead of tabs..." + "so_long"
+    variable = "Why the heck are spaces used instead of tabs..." + "so_long"
```
Comment (Mixih) [inline Users/Tutorial.rst]: Why was this deindented?

File: Users/Tutorial.rst [hunk 6]
```diff
@@ -387,3 +406,4 @@ due to its modular design:
    voice interaction) take a look at modules lying in `coalib/output <https://github.com/coala/coala/tree/master/coalib/output>`__.
 
 Happy coding!
+ng!
```
Comment (sims1253) [inline Users/Tutorial.rst]: what is this for?

File: Users/Tutorial.rst (hunk 5 above)
Comment (Udayan12167) [inline Users/Tutorial.rst]: Unrelated change.

**Comment by Mixih** [comment]
ack 2303ef8

**Comment by Mixih** [comment]
Waiting on rebase

**Comment by Mixih** [comment]
@rultor merge
